src/core_modules/capture-core/components/WidgetEnrollment/TransferModal/InfoBoxes/InfoBoxes.component.tsx

```diff
@@ -1,28 +1,27 @@
-// @flow
 import React from 'react';
 import cx from 'classnames';
-import { withStyles } from '@material-ui/core/styles';
+import { withStyles, type WithStyles } from '@material-ui/core/styles';
 import { colors, IconInfo16, IconWarning16 } from '@dhis2/ui';
 import i18n from '@dhis2/d2-i18n';
 import { useOrgUnitNameWithAncestors } from '../../../../metadataRetrieval/orgUnitName';
 import { OrgUnitScopes } from '../hooks/useTransferValidation';
 import { ProgramAccessLevels } from '../hooks/useProgramAccessLevel';
 
 type Props = {
-    ownerOrgUnitId: string,
-    validOrgUnitId: ?string,
-    programAccessLevel: string,
+    ownerOrgUnitId: string;
+    validOrgUnitId?: string;
+    programAccessLevel: string;
     orgUnitScopes: {
-        origin: $Keys<typeof OrgUnitScopes>,
-        destination: $Keys<typeof OrgUnitScopes>,
-    },
-    classes: Object,
+        origin: keyof typeof OrgUnitScopes | null;
+        destination: keyof typeof OrgUnitScopes | null;
+    };
+    classes: Record<string, string>;
```

Comment:
Since `WithStyles<typeof styles>` is used below on line 49, the `classes: Record<string, string>;` declaration is unnecessary and can be removed.